test/testnet/localnet/README.md

```diff
@@ -0,0 +1,94 @@
+# Remote Version of Localnet
+
+Remote version of the test network generated by running:
```

Comment:
I don't get it. `localnet` is a well established idea across all teams. Regardless if it's gaia or tendermint, `localnet` means a docker-based 4-node network brought up by docker-compose. Don't use localnet-start to start a remote network. Either call it remotenet-start or something else. (I think we did have remotenet-start a while ago.)